app/controllers/pre-payment.controller.js

```diff
@@ -24,6 +28,7 @@ function getContinueUrlForNewPaymentLinkJourney (product) {
 module.exports = (req, res) => {
   const product = req.product
   const correlationId = req.correlationId
+  const { reference, amount} = qs.parse(req.query)
```

Comment:
I'm not sure `qs` should be used here. 

It's not listed as a dependency of products-ui so I'm guessing it's a dependancy of a dependency which could be risky to rely on.

In fact it looks like the default Express 4.x [query parser](http://expressjs.com/en/api.html#app.settings.table) behaviour is to use a similar method to parse the string received with the request already.

I expect by the time we're accessing it on the `req` object it's already well formatted so the following should avoid any of these questions
```suggestion
  const { reference, amount} = req.query
```

Reply:
Thanks! I've updated